osu.Game/Screens/Menu/OnlineMenuBanner.cs

```diff
@@ -170,6 +173,8 @@ public partial class MenuImage : OsuClickableContainer
 
             private Sprite flash = null!;
 
+            public override bool IsPresent => base.IsPresent || Scheduler.HasPendingTasks;
```

Comment:
What's this for?

Reply:
It's a safety for

https://github.com/ppy/osu/blob/53b7c29488f6dd28ae30877e0df148d4c6be6d33/osu.Game/Screens/Menu/OnlineMenuBanner.cs#L214-L218

I will admit I spent over 1 hour making the test for this work. It turned out that online content was replacing the test content in headless only, amongst other issues. I'm not 100% sure this is required, but it feels like it may be for the an edge case where the banner somehow fades out before the delay is reached.